services/identity/service.go

```diff
@@ -39,14 +43,14 @@ func init() {
 // Service handles identities
 type Service struct {
 	*sda.ServiceProcessor
-	identities      map[string]*storage
+	Identities      map[string]*Storage
 	identitiesMutex sync.Mutex
```

Comment:
So you are sending a `Service` through the network ? It seems too late to change any design here but it seems strange... :s
And the mutex is needed because multiple client can ask the service the identities it holds at the same time ?